### ISSUE TITLE
Pod update failing for Xcodev6.4 -> Pod 0.37.2

### ISSUE THREAD

**Issue by pchhabrapfizer** [CocoaPods/CocoaPods#4951]
### Command

```
/usr/bin/pod install
```

### Report

* What did you do?

* What did you expect to happen?

* What happened instead?

### Stack

```
   CocoaPods : 0.37.2
        Ruby : ruby 2.0.0p481 (2014-05-08 revision 45883) [universal.x86_64-darwin14]
    RubyGems : 2.0.14
        Host : Mac OS X 10.10.5 (14F27)
       Xcode : 6.4 (6E35b)
         Git : git version 2.3.2 (Apple Git-55)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 2ccf448e83d956d041d9822ee2baa27a8b51a1b2
               pfizer - https://github.com/pfizer/Mobile-Reuse-PfizerPods.git @ 9bd06f3afa4aa8cab0befb96a8e84cf94f6ec074
```

### Plugins

```
cocoapods-plugins : 0.4.2
cocoapods-search  : 0.1.0
cocoapods-stats   : 0.6.2
cocoapods-trunk   : 0.6.4
cocoapods-try     : 0.4.5
```

### Podfile

```ruby
# Uncomment this line to define a global platform for your project
# platform :ios, "6.0"

target "PfizerEvents" do
    pod 'AFNetworking', '~> 1.3.0'
    pod 'MBProgressHUD', '~> 0.8'
    pod 'OHHTTPStubs', '~> 3.1.6'
    pod 'TCBlobDownload' ,' ~> 1.5.2'
    pod 'UrbanAirship-iOS-SDK'
    pod 'BlueCatsSDK', :git => 'https://github.com/bluecats/bluecats-ios-sdk.git', :tag => '0.3.2'

end
```

### Error

```
NoMethodError - undefined method `strip' for ["-ObjC", "-lz", "-lsqlite3"]:Array
/Library/Ruby/Gems/2.0.0/gems/xcodeproj-0.24.3/lib/xcodeproj/config/other_linker_flags_parser.rb:55:in `split'
/Library/Ruby/Gems/2.0.0/gems/xcodeproj-0.24.3/lib/xcodeproj/config/other_linker_flags_parser.rb:24:in `parse'
/Library/Ruby/Gems/2.0.0/gems/xcodeproj-0.24.3/lib/xcodeproj/config.rb:196:in `merge!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/generator/xcconfig/xcconfig_helper.rb:73:in `add_spec_build_settings_to_xcconfig'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/generator/xcconfig/xcconfig_helper.rb:53:in `block in add_settings_for_file_accessors_of_target'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/generator/xcconfig/xcconfig_helper.rb:52:in `each'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/generator/xcconfig/xcconfig_helper.rb:52:in `add_settings_for_file_accessors_of_target'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb:94:in `block in generate'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb:88:in `each'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb:88:in `generate'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb:36:in `save_as'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/installer/target_installer/aggregate_target_installer.rb:69:in `block in create_xcconfig_file'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/installer/target_installer/aggregate_target_installer.rb:66:in `each'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/installer/target_installer/aggregate_target_installer.rb:66:in `create_xcconfig_file'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/installer/target_installer/aggregate_target_installer.rb:16:in `block in install!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/user_interface.rb:140:in `message'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/installer/target_installer/aggregate_target_installer.rb:12:in `install!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/installer.rb:530:in `block (2 levels) in install_libraries'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/installer.rb:527:in `each'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/installer.rb:527:in `block in install_libraries'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/user_interface.rb:140:in `message'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/installer.rb:520:in `install_libraries'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/installer.rb:135:in `block in generate_pods_project'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/user_interface.rb:59:in `section'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/installer.rb:132:in `generate_pods_project'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/installer.rb:99:in `install!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/command/project.rb:101:in `run'
/Library/Ruby/Gems/2.0.0/gems/claide-0.8.1/lib/claide/command.rb:312:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/command.rb:46:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.2/bin/pod:44:in `<top (required)>'
/usr/bin/pod:23:in `load'
/usr/bin/pod:23:in `<main>'
```

**Comment by segiddins** [comment]
This ought to be fixed by the latest CocoaPods 1.0 beta.

**Comment by ryanwaggoner** [comment]
Just a note that I had this issue as well, and updating to latest beta did fix for me :+1:
